### PR TITLE
Database rename

### PR DESCRIPTION
**Pull request by Oliv751**

### CODE REVIEW

File: backend/src/models/consultantManager.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const AbstractManager = require("./AbstractManager");
```
Comment (rocambille) [inline backend/src/models/consultantManager.js:1]: this file is declaring a class, so file name should be in UpperCamelCase

**Review by rocambille — CHANGES_REQUESTED** [review]

File: backend/database.sql [hunk 2]
```diff
@@ -1,94 +1,202 @@
...
-  contract varchar(200) NOT NULL,
-  offer_description TEXT NOT NULL,
-  publication_date date,
-  company_id int(11) UNSIGNED NOT NULL,
-  CONSTRAINT fk_offer_company FOREIGN KEY (company_id) references company(id)
-)
-  engine=InnoDB DEFAULT charset=latin1;
+INSERT INTO
+    offer (
+        id,
```
Comment (Justine-Merlin) [inline backend/database.sql:132]: Same here.

File: backend/database.sql [hunk 3]
```diff
@@ -1,94 +1,202 @@
...
+        document_type varchar(100) NOT NULL,
+        date date NOT NULL,
+        document varchar(300) NOT NULL,
+        candidate_id int(11) UNSIGNED NOT NULL,
+        CONSTRAINT fk_document_candidate FOREIGN KEY (candidate_id) REFERENCES candidate(id)
+    ) engine = InnoDB DEFAULT charset = latin1;
 
+INSERT INTO
+    document (
+        id,
```
Comment (Justine-Merlin) [inline backend/database.sql:176]: Same here.

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: backend/database.sql (hunk 3 above)
Comment (Oliv751) [reply backend/database.sql]: all done , should be ok

**Comment by rocambille** [comment]
rename the files in a unix system (linux or mac) : windows is case insensitive :grin:

**Comment by Oliv751** [comment]
everything should be good now

**Pull request merged**